docs/source/getting_started/content/proc_starting-monitoring-alerting-telemetry.adoc

```diff
@@ -8,6 +8,9 @@ One of these is Prometheus and all the related monitoring, alerting, and telemet
 
 * A running {prod} virtual machine and a working [command]`oc` command.
 For more information, see <<accessing-the-openshift-cluster-with-oc_{context}>>.
+* Additional memory assigned to the {prod} virtual machine.
+At least 12 GiB of memory is recommended.
+For more information, see <<configuring-the-virtual-machine_{context}>>.
```

Comment:
@robin-owen `Additional memory need to be assigned to the {prod} virtual machine`  ? the current sentence seems like, it automatic assigned :)

Reply:
Updated and also added that the value for 12 GiB of memory is `12288`. Let's not make people do math if they don't have to. ;)